src/msw/listctrl.cpp

```diff
@@ -510,6 +515,11 @@ void wxListCtrl::SetWindowStyleFlag(long flag)
     }
 }
 
+bool wxListCtrl::IsDoubleBuffered() const
+{
+    return (GetHandle() && (ListView_GetExtendedListViewStyle(GetHandle()) & LVS_EX_DOUBLEBUFFER) != 0);
```

Comment:
We typically use `GetHWND()` or `GetHwnd()` macro in wxMSW code rather than the more generic `GetHandle()`.